config_defaults.yml

```diff
@@ -15,7 +15,7 @@
 Gemfile:
   supports_windows: false
   required:
-    ':development':
+    ':development, :unit_tests':
```

Comment:
What is the difference between development and unit tests? Or better asked, why two?